hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/package.json

```diff
@@ -35,7 +38,7 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "react-app-rewired --max_old_space_size=4096 build",
```

Comment:
is this removed because of the previous **build (8)** issue? did you check the history why it was there? maybe @adoroszlai do you know anything about this?

Reply:
No, sorry, I don't know anything about this.

Reply:
with removal, build was success as verified by smita